episodes/recipe_instructions.Rmd

```diff
@@ -13,7 +13,7 @@ exercises: 2
 ::::::::::::::::::::::::::::::::::::: objectives
 
 - Be able to prduce a batch of delicious cinnamon rolls
-- Understand how to adjust proooofing time
+- Understand how to adjust proving time
```

Comment:
Our contributing guidelines require the changes to be standard American English.  Proofing more commonly used in American English.
```suggestion
- Understand how to adjust proofing time
```